test/caching.jl

```diff
@@ -0,0 +1,25 @@
+println("------------------------------------")
+println("Caching tests")
+println("------------------------------------")
+
+# Tests for caching of Clebsch-Gordan coefficients
+import SUNRepresentations: cache_info, precompute_disk_cache, clear_disk_cache!
+
+# only remove cache if running on CI
+if get(ENV, "CI", false) == "true"
```

Comment:
Will there be a cache present when running on CI? And is it not automatically cleaned up afterwards? Or is this as a means of testing these functions and generate coverage?

Reply:
This is mostly to generate coverage, without affecting users.